tests/components/labOrderEntry/LabEntryForm.test.jsx

```diff
@@ -223,4 +229,10 @@ describe('Component: LabEntryForm', () => {
     });
     expect(component.find('PastOrders').exists()).toBeFalsy();
   });
+
+  it('should have lab orderables displayed in the category list', () => {
+    const component = getComponent();
+    const addButton = component.find('#category-button');
+    expect(addButton.length).toEqual(3);
```

Comment:
How is the add button used to test that the lab orderables display in a list?

Reply:
Sorry that was a mistake

Reply:
Corrected